scala-package/examples/src/main/scala/ml/dmlc/mxnetexamples/inferexample/SSDClassifierExample.scala

```diff
@@ -0,0 +1,96 @@
+package SSDClassifierExample
```

Comment:
Please add license and standard preamble docs as to what this does and how you use it.

Reply:
Added the License, will add a short description to this example